src/particles/beam/BeamParticleContainer.cpp

```diff
@@ -433,6 +436,7 @@ BeamParticleContainer::InSituComputeDiags (int islice, int islice_local)
     m_insitu_sum_rdata[14] += amrex::get<14>(a);
     m_insitu_sum_rdata[15] += amrex::get<15>(a);
     m_insitu_sum_rdata[16] += amrex::get<16>(a);
+    m_insitu_sum_rdata[17] += amrex::get<17>(a);
     m_insitu_sum_idata[ 0] += amrex::get<17>(a);
```

Comment:
Shouldn't this be next one?
```suggestion
    m_insitu_sum_idata[ 0] += amrex::get<18>(a);
```

Reply:
Yes